### DISCUSSION TITLE
Releases tab for package

### DISCUSSION THREAD

**Discussion by patrickdemooij9** [umbraco/Umbraco.Packages#117] [Ideas] [answered]
Usually, when I do an update for packages, I also add the release notes to the description so that people know what has actually changed. However, I feel that the description might not be the best place for this. Especially if you look at some older packages, you can see that there is a long list of updates (https://our.umbraco.com/packages/developer-tools/umbraco-forms/). That just makes the page a lot harder to read.

I think that adding a Releases tab somewhere would be a nice idea. The release tab would list all the different updates for the package. (maybe also have filters for Umbraco versions so that we can also switch between V7, V8, V9 releases). Each of those updates could then include the things that have changed and maybe also the Umbraco package for that update if you ever need to switch back.

**Comment by ronaldbarendse** [comment, marked as answer]
I've opened a new issue on the Marketplace tracker, since the issue is still valid: https://github.com/umbraco/Umbraco.Marketplace.Issues/issues/59.
